content/docs/ops/runbook/rotating-kubernetes.md

```diff
@@ -33,3 +54,21 @@ for pod in $(kubectl get pod | grep "kube2iam" | awk '{print $1}'); do kubectl d
 kubectl --namespace kube-system delete secret $(kubectl --namespace kube-system get secrets | grep "default-token" | awk '{print $1}')
 for pod in $(kubectl --namespace kube-system get pod | grep "fluentd-cloudwatch" | awk '{print $1}'); do kubectl --namespace kube-system delete pod ${pod}; done
 ```
+
+## Recreating Kubernetes DNS pods
+
+Rotating Consul certificates breaks the communication between the Consul proxy
+jobs on the Kubernetes `master` and `minion` VMs and the internal Kubernetes DNS
+pods. Fix Kubernetes DNS resolution with the following command.
+
+```sh
+kubectl get pods --namespace kube-system | grep kube-dns | awk '{ print $1 }' | xargs kubectl delete pods --namespace kube-system
+```
+
+Restart the proxy job in the `master` and `minion` Kubernetes BOSH deployment
+with the following commands.
```

Comment:
Mention that these are run inside of a jump box in the environment using the Bosh director.